src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -447,7 +451,11 @@ int fx_muxer_t::read_config_and_execute(
             }
             if (!found)
             {
-                trace::error(_X("Policy library either not found in deps [%s] or not found in %d probe paths."), deps_file.c_str(), probe_realpaths.size());
+                trace::error(_X("Expected hostpolicy lib either not found in deps.json [%s] or not found in %d probe paths: "), deps_file.c_str(), probe_realpaths.size());
```

Comment:
The "hostpolicy" library is an internal thing, let's be clearer about exactly what we're looking for and where. How about `Could not find required library 'libhostpolicy.so' in the dependencies list [deps_file], or the probing path [probing_path].` (replacing `libhostpolicy.so` with whatever the platform-specific library name is of course)